code/game/machinery/PDApainter.dm

```diff
@@ -103,6 +103,12 @@
 		stored_id_card = null
 		update_appearance(UPDATE_ICON)
 
+/obj/machinery/pdapainter/wrench_act(mob/living/user, obj/item/tool)
+	..()
```

Comment:
shouldn't these be `. = ..()` ?

Reply:
I've generally seen them use ..() so I standardized it to that, only saw 2-3 use . = ..() - I don't think it changes much though.

Reply:
it doesnt need to do . = ..() unless you want to pull the return value from the parent, which shouldnt need to as it should stop the act chain if it does something inside the parent iirc.

Reply:
parent's currently just an empty return. Parent's being called in case anything is added there in the future.

Reply:
unless there's an explicit reason not to, . = ..() is a good pattern. Makes future code changes simpler. 
on that topic, I think there is an expected return value, but only for pipes. God I hate pipe code.